### PR TITLE
Test helpers standardize format

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
Standardize formatting in TestHelpers code.

## Motivation and Context
Consistent format makes it easier for future readers/editors.

## How Has This Been Tested?
Running UI tests.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/TestHelpers/SharingHelper.php [hunk 1]
```diff
@@ -55,18 +55,19 @@ public static function createShare(
 		$linkName = null,
 		$expireDate = null,
 		$apiVersion = 1,
-		$sharingApiVersion = 1) {
+		$sharingApiVersion = 1)
+	{
```
Comment (individual-it) [inline tests/TestHelpers/SharingHelper.php]: according to PEAR standard it has to be `) {` in a new line for multi line function declaration

**Review by phil-davis — COMMENTED** [review]
Looks good to me.
It does not let me approve, because I am the originator of the PR.

**Comment by individual-it** [comment]
I think somebody else should also have a look into this formatting changes. They follow the owncloud code style patterns except the documentation says opening { should not be in a new line and does not make exceptions for functions.
Here we have placed { in new lines for functions, what I think is PEAR standard.

**Review by phil-davis — COMMENTED** [review]
Latest commits look good.

**Review by individual-it — APPROVED** [review]
Looks everything is now according to the coding styles in https://doc.owncloud.org/server/10.0/developer_manual/general/codingguidelines.html

**Pull request merged**